tests/transport/can/media/_socketcand.py

```diff
@@ -0,0 +1,153 @@
+import sys
+import typing
+import asyncio
+import logging
+import subprocess
+import pytest
+
+from pycyphal.transport import Timestamp
+from pycyphal.transport.can.media import Envelope, DataFrame, FrameFormat, FilterConfiguration
+from pycyphal.transport.can.media.socketcand import SocketcandMedia
+
+if sys.platform != "linux":  # pragma: no cover
+    pytest.skip("Socketcand test skipped because the system is not GNU/Linux", allow_module_level=True)
+
+
+GIBIBYTE = 1024**3
+
+MEMORY_LIMIT = 8 * GIBIBYTE
+"""
+The test suite artificially limits the amount of consumed memory in order to avoid triggering the OOM killer
+should a test go crazy and eat all memory.
+"""
```

Comment:
The resources will be configured for you by the top-level fixture. Please remove this and all usages below.